styles.css

```diff
@@ -657,6 +658,11 @@ textarea[id=subject] {
     cursor: pointer;
     mix-blend-mode: multiply;
     backdrop-filter: blur(6px);
+/* 
+left: 0px;
+right: 0px;
+
+bottom: 0px; */
```

Comment:
- [ ] [REQURED] Please, remove commented code. Because it polluted the codebase. It's better to use `git reset` to come back to previous versions ◀️.